cli/sync.go

```diff
@@ -118,6 +118,10 @@ var SyncMarkBadCmd = &cli.Command{
 		defer closer()
 		ctx := ReqContext(cctx)
 
+		if cctx.NArg() != 1 {
```

Comment:
you've duplicated the check just below here; move the check to the top and use `IncorrectNumArgsWithHint` to preserve the message